agent/testsuite-rhel7.sh

```diff
@@ -61,11 +61,10 @@ done
 ## Other integration tests ##
 TEST_LIST=(
     "test/test-exec-deserialization.py"
-#    "test/test-network/systemd-networkd-tests.py"
```

Comment:
Why the test is dropped, instead of uncomment the line?

Reply:
That's just for RHEL7 - I'm not sure if it's worth the hassle trying to backport all the systemd-networkd tests (and related stuff) all the way back to RHEL7's systemd version (but backporting for RHEL8 is already in progress: https://github.com/lnykryn/systemd-rhel/pull/287)